backend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gdx-agreements-tracker-api",
-  "version": "0.13.0",
+  "version": "0.14.0",
```

Comment:
not sure why you choose to change the version of the backend/package.json

Reply:
I figure if I'm deploying this to test, I might as well up the version